src/components/pages/devices/flyouts/SIMManagement/SIMManagement.js

```diff
@@ -5,8 +5,16 @@ import { Trans } from 'react-i18next';
 import { Link } from 'react-router-dom'
 
 import { permissions } from 'services/models';
-import { LinkedComponent } from 'utilities';
-import { FormControl, Protected } from 'components/shared';
+
+import { LinkedComponent, svgs } from 'utilities';
+
+import {
```

Comment:
@mechaffin I had to resolve conflicts. Can you make sure this section is still accurate?